spring/boot-webflux-autoconfigure/src/main/java/com/linecorp/armeria/spring/web/reactive/ArmeriaClientHttpConnector.java

```diff
@@ -119,7 +106,7 @@ private ArmeriaClientHttpRequest createRequest(HttpMethod method, URI uri) {
 
         final URI baseUri = URI.create(Strings.isNullOrEmpty(scheme) ? authority : scheme + "://" + authority);
         final HttpClientBuilder builder = new HttpClientBuilder(baseUri);
-        customizer.customize(builder);
+        configurators.forEach(c -> c.configure(builder));
```

Comment:
Should be `c -> c.customize(builder)`

Reply:
As we talked on LINE chat, let me remove the deprecated method `customize` because providing a default method is useless. :-)